content/en/users/check-in/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>404 Page not found</title>
+    <link rel="stylesheet" type="text/css" href="/scss/main.css">
```

Comment:
Ideally you shouldn't use an hardcoded path like here, as it will fail on things like the test instance.

Reply:
(this is something not ideal but I would be OK for merging despite this)

Reply:
I was meaning the deployed preview, the PR is prepended in the URL path, see https://docs.egi.eu/documentation/423/users/check-in. Thus any `/something` will break.
If there is no simple solution it's acceptable for such a page.